openshift/templates/pipeline/eagle-public-pipeline.bc.json

```diff
@@ -0,0 +1,88 @@
+{
+    "kind": "Template",
+    "apiVersion": "v1",
+    "metadata": {
+        "creationTimestamp": null,
+        "labels": {
+            "app": "${NAME}-pipeline",
```

Comment:
use $GROUP_NAME variable for app labels, as per [api-454](https://github.com/bcgov/eagle-api/pull/454#pullrequestreview-388658712)